src/game/Tactical/Merc_Hiring.cc

```diff
@@ -56,6 +56,16 @@ extern BOOLEAN gfFirstHeliRun;
 // Saved in general saved game structure
 INT16 g_merc_arrive_sector = START_SECTOR;
 
+void CreateSpecialItem(SOLDIERTYPE* const s, UINT16 item)
+{
+	OBJECTTYPE o;
+	CreateItem(item, 100, &o);
```

Comment:
Should we use the enum ITEMDEFINE?

Reply:
you mean for the param? Nothing uses it yet.

Reply:
I mean 100 can be replaced with the enum `LETTER` of from `enum ITEMDEFINE`

Reply:
I mean 100 can be replaced with the enum `LETTER` of from `enum ITEMDEFINE`

Reply:
No, that's the status percent.

Reply:
:facepalm: I see